docs/product-assembly_case-management-getting-started/src/main/asciidoc/case-management-itorders-roles-proc.adoc

```diff
@@ -9,21 +9,15 @@ Click image:3898.png[] to open the *Properties* menu on the right side of the de
 .ITOrders Case Roles
 image::case_roles.png[Case Roles]
 +
+<<<<<<< HEAD
 * `owner` - the person requesting the hardware. There can be only one of these roles.
```

Comment:
Is the owner role also set to cardinality 1? If so, I'd same the same as you noted for the manager. Just a late night thought.